charter.md

```diff
@@ -0,0 +1,34 @@
+#Charter for Working Group
+
+There are many situations in which it is desirable to share a digital credential with another person. For example, you may want to provide access to your vehicle to a Valet. You may also want to provide access to your home to your cat sitter. Or, you may want to share a hotel key with your spouse. Today, no such standardized method exists in a cross-platform, multidisciplinary capacity. 
+
+The WG charter includes the definition and standardization of a protocol that will facilitate such credential transfers from individual to individual. The protocol will leverage a “relay server” to transfer data from sender to recipient. The scope of the transfer is limited to a single origin device and a single destination device.
+
+Privacy goals include:
+
+- The relay server should not see sensitive details of the share
+- The relay server should not be able to provision the credential itself, acting as an intermediary for the recipient (MiTM)
+- The relay server should not know the identity of the sender nor receiver
+- Device OEMs should not be able to identity the social graph of whom has shared with whom
+
+Sufficient security measures should be embedded in the protocol in an effort to:
+
+- Ensure only the intended recipient is able to provision the credential
+- Ensure the credential can only be provisioned once (Anti-replay)
+- Ensure the sender has intent to share (secure user intent)
+- Allow the sender to revoke the credential, either before or after the recipient has accepted, where applicable
+
+The solution the WG comes up with must:
+
+- Allow a sender to initiate a share
+- Allow a recipient to view the share request, and provision the credential associated with the share upon receipt
```

Comment:
I think we should break this out a bit more. Maybe something like:

- Allow a recipient to view an open graph preview of a share
- Allow dynamic message formats based on the credential type
- Allow sender device and receiver device to quickly perform multiple round trip communications

Reply:
I would refrain from mentioning use of open graph for viewing in the charter document. Specific technologies can change in future.

Reply:
Agree with Yogesh here - the description should be as generic as possible, the concrete details to be defined in the draft itself.

Reply:
Makes sense. We can drop the specifics about how the preview is shown.